core/src/main/scala/kafka/admin/AdminClient.scala

```diff
@@ -115,6 +124,56 @@ class AdminClient(val time: Time,
     listAllGroupsFlattened.filter(_.protocolType == ConsumerProtocol.PROTOCOL_TYPE)
   }
 
+  class BrokerVersionInfo(val apiVersions: Map[ApiKeys, ApiVersion]) {
+    override def toString(): String = {
+      val bld = new StringBuilder()
+      var prefix = "\n  "
+      bld.append("{");
```

Comment:
That would be nice. Probably means that we'd want the tool to be in Java in the `tools` subproject. We would then not be able to use `AdminClient.scala`. Maybe we can do that as a subsequent JIRA as it will be easier once there is a Java `AdminClient`.